installer/windows-installer.iss

```diff
@@ -24,9 +24,10 @@
 ; NOTE: The value of AppId uniquely identifies this application.
 ; Do not use the same AppId value in installers for other applications.
 ; (To generate a new GUID, click Tools | Generate GUID inside the IDE.)
-AppId={code:GetAppId|4BB0DCDC-BC24-49EC-8937-72956C33A470}
+AppId={code:GetAppId|{{4BB0DCDC-BC24-49EC-8937-72956C33A470}}
 AppName=OpenShot Video Editor
 AppVersion={#VERSION}
+AppVerName={#MyAppName} {#VERSION}
```

Comment:
This updates the name of OpenShot displayed in the installer and Add/Remove Programs screens. "OpenShot Video Editor 3.0.0" for example.